core/src/scene/sceneLoader.cpp

```diff
@@ -408,6 +408,23 @@ void SceneLoader::loadStyleProps(Style* style, YAML::Node styleNode, Scene& scen
 
 }
 
+Node SceneLoader::propOr(const std::string& propStr, const Mixes& mixes) {
+
+    Node node;
+
+    for (const auto& mixNode: mixes) {
+        if (Node propNode = mixNode[propStr]) {
+            if (propNode.IsScalar()) {
+                if (propNode.as<std::string>() == "true") {
```

Comment:
YAML allows a few variations on Booleans (e.g. "true", "True", "TRUE") so for consistency we may want to use `as<bool>` here.